app/helpers/application_helper.rb

```diff
@@ -1,2 +1,5 @@
 module ApplicationHelper
+  def google_oauth_redirect_uri
+    ENV.fetch('GOOGLE_OAUTH_REDIRECT_URI') { 'http://localhost:3000/accounts/auth/google_oauth2' }
```

Comment:
Why not just use `"#{root_path}accounts/auth/google_oauth2"`?

Reply:
Can you redirect to a file path if you want a URI which is required by Google OAuth?

Reply:
We can use `Socket.gethostbyname(Socket.gethostname).first` in production. See: https://stackoverflow.com/questions/151545/how-can-i-get-the-fqdn-of-the-current-host-in-ruby

We can use `localhost:3000` when we're in development.